yarn-project/sequencer-client/src/sequencer/processed_tx.ts

```diff
@@ -104,6 +105,7 @@ export function makeEmptyProcessedTx(
     unencryptedLogs: new TxL2Logs([]),
     data: emptyKernelOutput,
     proof: emptyProof,
+    newContracts: [ExtendedContractData.random()],
```

Comment:
Let's use `empty` instead of `random` here for consistency

Reply:
I'll add an `.empty` static method to the class 👍